code/game/objects/items/storage/holsters.dm

```diff
@@ -28,11 +29,36 @@
 		/obj/item/gun/energy/disabler,
 		/obj/item/gun/energy/dueling,
 		/obj/item/food/grown/banana,
+		/obj/item/gun/energy/laser/thermal
 		))
 
+/obj/item/storage/belt/holster/thermal
+	name = "thermal shoulder holsters"
+	desc = "A rather plain pair of shoulder holsters with a bit of insulated padding inside. Meant to hold a twinned pair of thermal pistols, but can fit several kinds of energy handguns as well."
+
+/obj/item/storage/belt/holster/thermal/ComponentInitialize()
+	. = ..()
+	var/datum/component/storage/STR = GetComponent(/datum/component/storage)
+	STR.max_items = 2
+	STR.max_w_class = WEIGHT_CLASS_NORMAL
+	STR.set_holdable(list(
```

Comment:
why can these hold energy pistols (and even bananas) but not normal revolvers?

Reply:
_shrug_